.github/issue_template/config.yml

```diff
@@ -0,0 +1,20 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Issue with .NET SDK
```

Comment:
Also add lines for the runtime itself (missing from what you grabbed since it was in runtime repo)
  https://github.com/dotnet/wpf/blob/6a3e661716c27aa8f551448742df6f9b673cf0a2/.github/ISSUE_TEMPLATE/config.yml#L3